assignment2/q1.py

```diff
@@ -0,0 +1,36 @@
+# Print AncestorsGiven a Binary Tree and a key, 
+# write a function that prints all the ancestors of the key in the givenbinary tree.
+class Node:
+ 
+    
+    def __init__(self, data):
+        self.data = data 
+        self.left = None
+        self.right = None
+
+    def printAncestors(root, target):
+
```

Comment:
It's possible `printAncestors` was a static function in an earlier version (not commited) version of the code? And the test code below might have not been updated (as you point out)?